pkg/json/v0/config/tasks.json

```diff
@@ -105,5 +106,78 @@
       "title": "Output",
       "type": "object"
     }
+  },
+  "TASK_JQ": {
+    "instillShortDescription": "Process JSON through a jq command",
+    "input": {
+      "description": "Source JSON and jq command",
+      "instillUIOrder": 0,
+      "properties": {
+        "jsonInput": {
+          "instillUIOrder": 0,
+          "description": "JSON string to be processed",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "instillUIMultiline": true,
+          "title": "JSON input",
+          "type": "string"
+        },
+        "jqFilter": {
+          "instillUIOrder": 1,
+          "description": "Filter, in jq syntax, that will be applied to the JSON input",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "instillUIMultiline": true,
+          "title": "JQ filter",
+          "type": "string"
+        }
+      },
+      "required": [
+        "jsonInput",
+        "jqFilter"
+      ],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "description": "Results of the jq command applied to the input JSON",
+      "instillEditOnNodeFields": [
```

Comment:
`instillEditOnNodeFields` is not actually needed in the output schema; it can be removed. (Other connectors/operators have the same issue; we can fix them later.)